app/models/lockstep/user.rb

```diff
@@ -3,6 +3,15 @@ class Lockstep::User < Lockstep::ApiRecord
   self.id_ref = "user_id"
   load_schema(Schema::UserAccount)
 
+  def self.invite(email_addresses)
+    emails = email_addresses.map { |email| { email: email } }
+    resp = resource.post('/invite', body: emails, params: {})
+    parsed_response = JSON.parse(resp.body)
+    raise Lockstep::Exceptions::BadRequestError, parsed_response['title'] unless resp.code == '200'
```

Comment:
We cannot throw a bad request exception for non-200 response. We need to handle every status code correctly. This implementation should be similar to `ApiRecord#bulk_import` method.

Reply:
Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/dcc473c36d201f48d2d878ce0aa2ad6caf42ff01)